pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -1036,4 +1036,8 @@ const (
 	// LabelInjectGardenKubeconfig is a constant for a label on workload resources that indicates that a kubeconfig to
 	// the garden cluster should be injected.
 	LabelInjectGardenKubeconfig = "extensions.gardener.cloud/inject-garden-kubeconfig"
+
+	// AnnotationDisableShootReconciliations is the key for the emergency switch annotation for the seed resource
+	// to temporarily pause further shoot reconciliations.
+	AnnotationDisableShootReconciliations = "gardener.cloud/disable-shoot-reconciliations"
```

Comment:
Should we call it a bit differently to raise awareness that this is not a 0-8-15 annotation?

Something like `shoot.gardener.cloud/emergency-stop-reconciliations`? :)